semgrep-core/src/analyzing/Dataflow_tainting.ml

```diff
@@ -41,6 +41,9 @@ module VarMap = Dataflow.VarMap
 type mapping = unit Dataflow.mapping
 (*e: type [[Dataflow_tainting.mapping]] *)
 
+(* Tracks tainted functions. *)
+type fun_env = (Dataflow.var, unit) Hashtbl.t
```

Comment:
In this case we are converting the `IL.name` to a string "<source name>:<sid>", I clarified that in a comment. I did it this way because it's how the `Dataflow.mapping`s work, but we could also have a proper key type `string * AST_generic.sid`.